nikola/plugins/compile/rest/__init__.py

```diff
@@ -177,11 +178,15 @@ class NikolaReader(docutils.readers.standalone.Reader):
     def __init__(self, *args, **kwargs):
         """Initialize the reader."""
         self.transforms = kwargs.pop('transforms', [])
-        docutils.readers.standalone.Reader.__init__(self, *args, **kwargs)
+        self.no_title_transform = kwargs.pop('no_title_transform', False)
+        super(NikolaReader, self).__init__(*args, **kwargs)
```

Comment:
I think you forgot `self` as the first argument to the call to `super`'s `__init__`.